infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-sidecar.conf

```diff
@@ -4,8 +4,6 @@
     Path               /var/log/containers/${HOSTNAME}*.log
     Exclude_Path       /var/log/containers/${HOSTNAME}*${FLUENT_CONTAINER_NAME}*.log
     Parser             docker
-    DB                 /var/log/flb_tail_input_${HOSTNAME}.db
```

Comment:
A quoi correspond ce changement ?

Reply:
Ce type de config affiche souvent des erreurs dans le pod fluent-bit de type (On n'utilise pas de bdd sqlite):
[2022/06/10 14:22:46] [error] [sqldb] cannot open database /var/log/flb_kube.db
[2022/06/10 14:22:46] [error] [input:tail:tail.0] could not open/create database

Maintenant avec les nouvelles versions de fluent-bit, le pod crash à cause de cette config (ce qui n'était pas le cas avec les anciennes versions malgré l'erreur).